src/Core/src/Platform/Android/RadioButtonExtensions.cs

```diff
@@ -17,7 +17,12 @@ public static void UpdateIsChecked(this AppCompatRadioButton platformRadioButton
 
 		public static void UpdateContent(this AppCompatRadioButton platformRadioButton, IRadioButton radioButton)
 		{
-			platformRadioButton.Text = $"{radioButton.Content}";
+			if (radioButton.GetWindow().FlowDirection == FlowDirection.RightToLeft)
```

Comment:
Even if a flow direction check is necessary here (and I doubt that), it should be a check against the flow direction of the parent, not the window. It's possible that one of the layers of the hierarchy between the RadioButton and the Window has set an explicit FlowDirection that doesn't match the Window's FlowDirection.